core-web/libs/utils-testing/src/lib/dot-containers.mock.ts

```diff
@@ -74,5 +74,6 @@ export const dotContainerMapMock = (): DotContainerMap => {
 
 export const containersMock: DotContainer[] = Array.from(Array(10).keys()).map((number) => ({
     friendlyName: `Container ${number}`,
-    parentPermissionable: { hostname: '' }
+    parentPermissionable: { hostname: '' },
+    identifier: String(number)
 }));
```

Comment:
I think this is not needed, you can create a mock on `mocks.ts` inside the library